tests/system/base.py

```diff
@@ -28,6 +28,7 @@ class TestMLRunSystem:
         "V3IO_FRAMESD",
         "V3IO_USERNAME",
         "V3IO_ACCESS_KEY",
+        "DEFAULT_SPARK_SERVICE",
```

Comment:
In order for this to be set in the system tests CI you'll need to get it from the secret (I'll create it) [here](https://github.com/mlrun/mlrun/blob/development/.github/workflows/system-tests-enterprise.yml#L140) - and "bubble it down" all the way to [here](https://github.com/mlrun/mlrun/blob/development/automation/system_test/prepare.py#L70)
2. Add it here -